core/src/main/scala/cromwell/core/labels/Label.scala

```diff
@@ -5,26 +5,37 @@ import cats.data.Validated._
 import cats.syntax.cartesian._
 import cats.syntax.validated._
 
+import scala.util.matching.Regex
+
 sealed abstract case class Label(key: String, value: String)
 
 object Label {
 
   // Yes, 63. Not a typo for 64.
   // See 'labels' in https://cloud.google.com/genomics/reference/rpc/google.genomics.v1alpha2#google.genomics.v1alpha2.RunPipelineArgs
   private val MaxLabelLength = 63
-  val LabelRegexPattern = "[a-z]([-a-z0-9]*[a-z0-9])?"
+  val GoogleLabelRegexPattern = "[a-z]([-a-z0-9]*[a-z0-9])?"
```

Comment:
I agree w/ @mcovarr here - backend specific stuff should be in the backend package